rethinkdb/rethinkdb.d.ts

```diff
@@ -36,10 +36,15 @@ declare module "rethinkdb" {
     export class Cursor {
         hasNext(): boolean;
         each(cb: (err: Error, row: any) => void, done?: () => void): void;
+        each<T>(cb: (err: Error, row: T) => void, done?: () => void): void;
```

Comment:
this `T` cannot be inferred from anything unless user has provided it via type annotation on parameter which means that user has to either write type argument explicitly `each<Foo>((err, v) => ...)` or specify type annotation on the parameter: `each((err, v: Foo) => ...)`. Just curious is this the intended behavior?